readthedocs/oauth/notifications.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+from __future__ import division, print_function, unicode_literals
+
+from readthedocs.notifications import Notification
+from readthedocs.notifications.constants import HTML, WARNING, INFO
+
+
+class AttachWebhookNotification(Notification):
+
+    name = 'attach_webhook'
+    context_object_name = 'provider'
+    subject_success = 'Attach webhook success'
+    subject_fail = 'Attach webhook failed'
```

Comment:
I'm mostly certain that subject is used for email only, so these should be required. Double check that I'm not incorrect here.

Also, for purposes of maintaining good UX, copy for this email subject (if we needed to send one) should be:
`Attach webhook success` -> `Webhook successfully configured`
`Attach webhook failed` -> `Webhook could not be configured`

These would be clearer to the user -- they are speaking in core developer language currently.